src/structures/Guild.js

```diff
@@ -534,6 +534,30 @@ class Guild extends AnonymousGuild {
     );
   }
 
+  /**
+   * The maximum bitrate available for this guild
+   * @type {number}
+   * @readonly
+   */
+  get maximumBitrate() {
+    if (this.features.includes('VIP_REGIONS')) {
+      return 384000;
+    } 
+    
+    switch (PremiumTiers[this.premiumTier]) {
+      case 0:
+        return 96000;
```

Comment:
Wouldn't it be better to have a proper enum for that instead of leaving magic numbers in the middle of the code?

Reply:
> Wouldn't it be better to have a proper enum for that instead of leaving magic numbers in the middle of the code?

It could be a better idea. If you all prefer to go with this option, i can do it.

Reply:
I don't think an enum is the way to go because these are just maximum values and bitrates can have all sorts of values.
Either way did you test this? Are you 100% sure this works?

Reply:
Yes, it is! Tested!